source/index.md

```diff
@@ -568,26 +566,76 @@ otherwise, exactly one payment method must be used.
 
 ##### Method: `applepay`
 
-<dl class="dl-horizontal">
-  <dt>applepay[token]</dt>
-  <dd>[:json:] <br /> Full, raw <code>PKPaymentToken</code> object, UTF-8 encoded serialization of a JSON dictionary.</dd>
-  <dt>applepay[symmetric_key]</dt>
-  <dd>[:hex:] <br /> EC symmetric AES key that can decrypt <code>data</code>
-      from the <code>PKPaymentToken</code>; hex formatted as it is a SHA256.</dd>
-</dl>
+Apple Pay requires data of the payment token to be decrypted. For this, the
+merchant's private key is needed to derive a symmetric key which is then used to
+decrypt the PAN, expiry etc.
```

Comment:
Suggestion: s/to decrypt the PAN, expiry etc/for the decryption/

Reply:
I like it. What about
```
diff --git a/source/index.md b/source/index.md
index 0ce2c8b..f236698 100644
--- a/source/index.md
+++ b/source/index.md
@@ -566,9 +566,9 @@ method must be used.

 ##### Method: `applepay`

-Apple Pay requires data of the payment token to be decrypted. For this, the
-merchant's private key is needed to derive a symmetric key which is then used to
-decrypt the PAN, expiry etc.
+Apple Pay requires the payment details (PAN, expiry, etc.) of the payment token
+to be decrypted. For this, the merchant's private key is needed to derive a
+symmetric key which is then used for decryption.
```
?

Reply:
👍